backend/lib/endpoints/schema/posts.js

```diff
@@ -1,53 +1,94 @@
 const S = require("fluent-schema");
 const { strictSchema } = require("./utils");
 
+const { join } = require("path");
+const PATH_TO_SCHEMAS = join(__dirname, "../../models/schemas/v2");
+const { schema: postSchema } = require(join(PATH_TO_SCHEMAS, "post"));
+const { schema: orgSchema } = require(join(PATH_TO_SCHEMAS, "individualUser"));
+const { schema: userSchema } = require(join(
+  PATH_TO_SCHEMAS,
+  "organizationUser",
+));
+
+const EXPIRATION_OPTIONS = ["day", "week", "month", "forever"];
```

Comment:
Should we create something like a json file to store constants and enum data? I'm thinking this may help collaborate with the frontend. What do you think?

Reply:
Well, I agree that we should have only one source for them. 

Initially, I was thinking of using it at the model, accessing it through the schema as I did with the visibility options and etc, and then adding there would update everything. But that covers only what we store at the database, cases like this wouldn't be covered (because the interface options are periods and we store the expire date). 

How do you think we can mix those things?
Maybe the JSON defines all options (e.g. `posts.json` would cover DB and interface fields for posts) and at the models, we would use that to define the field enums?

Reply:
Yup I agree - having a JSON that defines all the options that are available for the models and frontend. Then we can import them to models and frontend components wherever needed. Does that work or does it sound to simple to work?

Reply:
Yeah great idea to have a shared file, this is often done with JSON schema's, sometimes for both API validation, database schema's and to share it with the frontend. This is also similar to GraphQL types... but in this case I think it makes sense to have a file that the frontend could use as well, instead of mixing DB and API fields into one file and then use that at the model?

Reply:
Well, we have a J[SON Schema already for the DB fields](https://github.com/FightPandemics/DataModel/blob/master/v2/scripts/JSON_Schema/applicationDB/posts.json) since Mongo uses that. 

But how would accessing the enums through the JSOM schema be any different than accessing it through the schema?

Reply:
Well I think the main advantage here can be DRY, avoiding duplication, by having a single source of truth for the values that can be used for certain fields.
So for that matter we could consider to move these to a shared file.
The question is, do we share them with mongo schema & API schema, and also with frontend?

I think for us Mongo schema's and fluent schema's are always made manually, but either you could export things from the model files separately and import it in the API schema's (just JS literal for example for the enums), or a separate file for example in `common/schema` could also export named enums etc.
 
Sharing with the frontend is a little bit more complex